docs/modules/provisioning-app/pages/cd-user.adoc

```diff
@@ -0,0 +1,17 @@
+:toc: macro
+
+= Syncing your Technical User With ODS Components
+
+When you create a new project, the provisioning app triggers a Jenkins pipeline. This pipeline adds `CD_USER_ID` as the name and `CD_USER_B64` as the encoded password to the secrets of your newly generated Openshift cd project. This is the technical user for your project. Every time a (cd) project is generated, these parameters are added to the secrets of your Openshift project. At this point however, you're still not ready to push some code into your repository or succesfully generate a new quickstarter using ods-shared-library. In order to achieve the two, you would manually need to add this same technical user to the repository settings of the two ODS repositories: `ods-jenkins-shared-library` and `ods-quickstarters`. Hence you have a technical user that will have read access to your project. 
+
+== Setting Your Project Specific CD User
+
+Keep in mind that at this point, we assume you've already created at least two ODS projects that use the same technical user and you'd like to change one of them. When you make this change, you need to have this new technical user be in sync in three places:
```

Comment:
Why two? One is enough?

Reply:
The reason why I said two was because the second project using the same credentials as the first one being a deterrent to the developer. It wasn't clear from here that I was encouraging the developer to use separate cd user credentials for 1+...

Reply:
It makes sense to strongly encourage the developer not to use global cd user as you pointed out